### ISSUE TITLE
sorting by [range]

### ISSUE THREAD

**Issue by buseca** [dbertella/relist#79]
current sorting by range results in unexpected things, for example if I use a descending order, the 2-3 range comes before the 1-5.

I think we should consider this logic instead:

- If the users select ASCENDING, it means they are interested in seeing lower ranges first. Hence we should take the **low** limit of the range and sort by it.
- If the they select DESCENDING, they are interested in the higher ranges. Hence we should sort considering the **high** limit of the ranges.

![Image](https://github.com/dbertella/relist/assets/3997836/963df108-8748-46d4-95ee-7f9cc1e29892)

**Comment by buseca** [comment]
@dbertella what you think?

**Comment by dbertella** [comment]
Yeah I see it's a bit of a pain. I don't get your logic though. Are you saying that if I have 1-5 and I sort by ascending order we should use 1 or 5?
I think you should take 1 for ascending and 5 for descending. But yeah let me check the current logic and I can fix it

**Comment by buseca** [comment]
Right, I mistaken the description.
Ascending --> low limit range
Descending --> upper limit range
